ISSUE_SORTING.md

```diff
@@ -1,21 +1,39 @@
 ## The Issue Sorting Process at `conda`
 
+> **Note:**
+> This process is similar to that of "triaging" but we've chosen to use different terminology because "triaging" is a word related to very weighty topics (*i.e.*, injuries and war) and we would like to be sensitive to those connotations. Additionally, we are taking a more "fuzzy" approach, *e.g.*, severities may not be assigned, etc.
```

Comment:
```suggestion
> This process is similar to that of "triaging", but we've chosen to use different terminology because "triaging" is a word related to very weighty topics (*i.e.*, injuries and war) and we would like to be sensitive to those connotations. Additionally, we are taking a more "fuzzy" approach to sorting (*e.g.*, severities may not be assigned, etc.).
```